projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrService.java

```diff
@@ -848,6 +816,8 @@ public JSONArray getQuestionTemplates(@FormDataParam(CoordConsts.SVC_KEY_API_KEY
 
       checkStringParam(apiKey, "API key");
 
+      checkApiKeyValidity(apiKey);
```

Comment:
One PR should have one purpose only. Please fix the existing API key issue in a separate PR.

Reply:
done